app/views/shared/_nav.html.slim

```diff
@@ -15,13 +15,15 @@ nav.navbar.navbar-inverse
               span.glyphicon.glyphicon-plus aria-hidden='true'
               span.caret
             ul.dropdown-menu role='menu'
+              li= link_to('Upload Resource', new_resource_path)
```

Comment:
Can we change this to say 'Add Resource'? I feel like "Upload" will indicate that we're uploading a file, when in reality we're just linking to a URL.